src/geo.c

```diff
@@ -241,6 +245,7 @@ int geo_bios_load_mem(void *data, size_t size) {
     return geo_bios_load(&zip_archive);
 }
 
+#ifndef __LIBRETRO__
```

Comment:
Is there a reason this needs to be left out of the build?
Please refer to the "Contributing" section of the README. This sort of thing should be avoided in favour of generic solutions.

Reply:
I do not know how the core is meant to work standalone, outside of libretro: if you don't have access to the VFS functionality you may as well continue to keep using `miniz`'s functionmality to load archives from files directly instead of buffering to memory through the VFS

Reply:
In this case, let's just leave it how it is now. There shouldn't be any harm in keeping this function in the build.